docs/intro.rst

```diff
@@ -489,25 +489,25 @@ The two tabs below provide the two formulations.
         The model in this tab is formulated using the implicitly 
         constrained :class:`~dwave.optimization.symbols.List` symbol. 
  
- 	>>> model = Model()
- 	>>> # Add the constants
-	>>> cost = model.constant(cost_per_day)
-	>>> distances = model.constant(distance_matrix)
-	>>> # Add the decision symbol
-	>>> route = model.list(4)
-	>>> # Optimize the objective
-	>>> model.minimize((cost * distances[route[:-1],route[1:]]).sum())
-	
-	You can see the objective values for the least and most costly routes 
-	as permutations of the :math:`[0, 1, 2, 3]` list as follows:
-	
-	>>> with model.lock():
-	...     model.states.resize(2)
-	...     route.set_state(0, [3, 2, 1, 0])
-	...     route.set_state(1, [2, 1, 3, 0])
-	...     print(int(model.objective.state(0)), int(model.objective.state(1)))
-	14 36
-	
+        >>> model = Model()
+        >>> # Add the constants
+        >>> cost = model.constant(cost_per_day)
+        >>> distances = model.constant(distance_matrix)
+        >>> # Add the decision symbol
+        >>> route = model.list(4)
+        >>> # Optimize the objective
+            >>> model.minimize((cost * distances[route[:-1],route[1:]]).sum())
```

Comment:
This line doesn't look right though it renders okay

Reply:
oops, missed that one. Will fix